### PR TITLE
(DOCSP-27576): Flutter Device Sync tutorial fixes

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Flutter device sync tutorial fixes

### Jira

- https://jira.mongodb.org/browse/DOCSP-27576

### Staged Changes

- [Flutter Device Sync tutorial](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/DOCSP-27576/tutorial/flutter/)

### Reminder Checklist

You might need to also update some corresponding pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-realm update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/tutorial/flutter.txt [hunk 1]
```diff
@@ -187,6 +194,12 @@ Set up the Template App
       in either the Android emulator, iOS Simulator, physical mobile device,
       or desktop emulator.
 
+      First install all dependencies by running the following in your terminal:
+
+      .. code-block:: sh
+
+         flutter pub get
+
       Attach to a device and run the Flutter application.
```
Comment (mongodben) [inline source/tutorial/flutter.txt:203]:
```suggestion
      Then, attach to a device and run the Flutter application.
```

File: source/tutorial/flutter.txt [hunk 2]
```diff
@@ -128,7 +132,10 @@ on it.
       If you use this process to get the client code, you must create a
       template app to use with the client. Follow the instructions at 
       :ref:`create-template-app` to use the Atlas App Services UI, {+cli+},
-      or Admin API to create a Device Sync template app.
+      or Admin API to create a Device Sync backend template app.
+
+      Clone the repo and following the instructions in the :file:`README` to add
```
Comment (dacharyc) [inline source/tutorial/flutter.txt:137]:
```suggestion
      Clone the repo and follow the instructions in the :file:`README` to add
```

File: source/tutorial/flutter.txt [hunk 3]
```diff
@@ -187,7 +194,13 @@ Set up the Template App
       in either the Android emulator, iOS Simulator, physical mobile device,
       or desktop emulator.
 
-      Attach to a device and run the Flutter application.
+      First install all dependencies by running the following in your terminal:
```
Comment (dacharyc) [inline source/tutorial/flutter.txt:197]:
```suggestion
      First, install all dependencies by running the following in your terminal:
```

**Review by dacharyc — APPROVED** [review]
Couple of minor suggestions, but LGTM 👍

**Pull request merged**